public/core.d.ts

```diff
@@ -4596,6 +4596,94 @@ declare namespace JMap {
        */
       function removeByIds(basemapIds: string[]): void
     }
+
+    namespace Attribution {
+      /**
+       * ***JMap.Map.Attribution.getAll***
+       * 
+       * Returns all the attributions displayed on the map.
+       * 
+       * @example ```ts
+       * 
+       * // Get all the attributions currently displayed.
+       * const attributions: JMapAttribution = JMap.Map.Attribution.getAll()
+       * 
+       * ```
+       */
+      function getAll(): JMapAttribution[]
+
+      /**
+       * ***JMap.Map.Attribution.addSingle***
+       * 
+       * Add a custom attribution on the map.
+       * 
+       * @throws Errors if incorrect or taken id, if both text and imgSrc are not a string, if href is initialized but not a string
```

Comment:
Errors if some parameters ar invalid, or if an attribution having the same id already exists